ADOPTERS.md

```diff
@@ -0,0 +1,32 @@
+# Adopters
+
+This list captures the set of organizations that are using Crossplane within their environments. If
+you are an adopter of Crossplane and not yet on this list, we encourage you to add your organization
+here as well!
+
+The goal for this list is to be the complete and authoritative source for the entire community of
+Crossplane adopters, and give inspiration to others that are earlier in their Crossplane journey.
+
+Contributing to this list is a small effort that has a **big impact** to the projects growth,
+maturity, and momentum.  Thank you to all adopters and contributors of the Crossplane project!
+
+## Updating this list
+
+To add your organization to this list, you can choose any of the following options:
+
+1. [Open a PR](https://github.com/crossplane/crossplane/pulls) to directly update this list, or
+   [edit this file](https://github.com/crossplane/crossplane/edit/master/ADOPTERS.md) directly in
+   Github
+1. Fill out the [adopters form](https://forms.gle/dBQhiyYkYSdzXovN6)
+1. Send an email to <jared@upbound.io> with your information
```

Comment:
Maybe it would help to specify what information is required?

Reply:
Do we have a steering committee mailing list? Just thinking about bus-factor here.

Reply:
good points @Piotr1215 and @negz - i updated to be explicit about providing the information in the format of the table below, and we're not using the steering committee email list that I had forgotten about :)